src/lib.rs

```diff
@@ -1690,4 +1713,39 @@ mod tests {
         assert_ne!(map_a, map_c);
         assert_ne!(map_c, map_a);
     }
+
+
+    #[test]
+    fn drain_basic() {
+        let mut map_a = OrderMap::new();
```

Comment:
(Not very actionable comment)

Quickcheck tests are always better than example tests. In this case we reuse a solid std iterator and we don't really *need* any particular test.